src/utils/smart-notifier-handler.js

```diff
@@ -90,17 +91,17 @@ function processHandlersFactory(handlers, userHandlerOptions) {
   };
 }
 
-
-module.exports = function smartNotifierHandler({ handlers = {}, userHandlerOptions = {} }) {
+module.exports = function smartNotifierHandler({ handlers = {}, options = {} }) {
```

Comment:
This is the line which breaks from:
`{ handlers = {}, userHandlerOptions = {} }`
to
`{ handlers = {}, options = {} }`